Services/TermsOfService/templates/default/tpl.withdraw_terms_of_service.html

```diff
@@ -0,0 +1,17 @@
+<table class="fullwidth">
```

Comment:
Do we really need to introduce a new HTML template here?

If so, please use placeholders for the commands and set the corresponding variables for these commands in PHP.